frontend/src/components/ShootVersion/GShootVersionChip.vue

```diff
@@ -105,11 +105,15 @@ SPDX-License-Identifier: Apache-2.0
             />
           </template>
           <g-list-item-content label="Update Information">
-            <div v-if="shootSupportedPatchAvailable">
-              Patch is available for this version
-            </div>
-            <div v-if="shootSupportedUpgradeAvailable">
-              Upgrade is available for this version
+            <div :class="{ 'list-style': shootSupportedPatchAvailable && shootSupportedUpgradeAvailable }">
```

Comment:
If both boolean flags are false we render an empty list item content block.

Reply:
this cannot happen as the whole section is hidden in tis case see line 98